src/main/java/duke/command/DeleteCommand.java

```diff
@@ -0,0 +1,42 @@
+package duke.command;
+
+import duke.exceptions.DukeIndexException;
+import duke.util.TaskList;
+import duke.util.Ui;
+import duke.util.Storage;
+
+import java.util.ArrayList;
+
+/**
+ * DeleteCommand class which handles the logic of deleting tasks.
+ */
+public class DeleteCommand extends Command {
+    private final String fullCommand;
```

Comment:
You can consider adding Javadocs for class members as stated under Java coding standards. 😄 
![image](https://user-images.githubusercontent.com/77244987/131902924-e9151807-78c5-4c9d-a77d-e7f8036faf37.png)